src/openloco/windows/station_list.cpp

```diff
@@ -25,16 +38,138 @@ namespace openloco::ui::windows::station_list
+        make_widget( {494, 43}, {120, 12}, widget_type::wt_14, 1, string_ids::null, string_ids::tooltip_sort_by_cargo_accepted),
+        make_widget( {3, 56}, {594, 126}, widget_type::scrollview, 1, scrollbars::vertical),
+        widget_end(),
+    };
+
+    static window_event_list _events;
+
+    static void event_08(window* window);
+    static void event_09(window* window);
+    // static void cursor(window* window);
+    static void draw(ui::window* window, gfx::drawpixelinfo_t* dpi);
+    // static void draw_scroll(ui::window* window, gfx::drawpixelinfo_t* dpi, uint32_t scrollIndex);
+    static void get_scroll_size(ui::window* window, uint32_t scrollIndex, uint16_t* scrollWidth, uint16_t* scrollHeight);
+    static void on_dropdown(ui::window* window, widget_index widgetIndex, int16_t itemIndex);
+    static void on_mouse_down(ui::window* window, widget_index widgetIndex);
+    static void on_mouse_up(ui::window* window, widget_index widgetIndex);
+    // static void on_scroll_mouse_down(ui::window* window, int16_t x, int16_t y, uint8_t scroll_index);
+    // static void on_scroll_mouse_over(ui::window* window, int16_t x, int16_t y, uint8_t scroll_index);
+    static void on_update(window* window);
+    static void prepare_draw(ui::window* window);
+    static void tooltip(ui::window* window, widget_index widgetIndex);
+
+    static void init_events()
+    {
+        _events.event_08 = event_08;
+        _events.event_09 = event_09;
+        // _events.cursor = cursor;
+        _events.draw = draw;
+        // _events.draw_scroll = draw_scroll;
+        _events.get_scroll_size = get_scroll_size;
+        _events.on_dropdown = on_dropdown;
+        _events.on_mouse_down = on_mouse_down;
+        _events.on_mouse_up = on_mouse_up;
+        _events.on_update = on_update;
+        // _events.scroll_mouse_down = on_scroll_mouse_down;
+        // _events.scroll_mouse_over = on_scroll_mouse_over;
+        _events.prepare_draw = prepare_draw;
+        _events.tooltip = tooltip;
+    }
+
+    // 0x004910E8
+    static void sub_4910E8(window* window)
+    {
+        // This sub is only called from events in this window.
+        registers regs;
+        regs.esi = (int32_t)window;
+        call(0x004910E8, regs);
+    }
+
+    // 0x00490F6C
+    window* open(uint16_t companyId)
+    {
+        window* window = WindowManager::bringToFront(WindowType::stationList, 0);
+        if (window != nullptr)
+        {
+            if (input::is_tool_active(window->type, window->number))
+                input::cancel_tool();
+
+            // Still active?
+            window = WindowManager::bringToFront(WindowType::stationList, 0);
+        }
+
+        if (window == nullptr)
+        {
+            // 0x00491010
+            window = WindowManager::createWindow(
+                WindowType::stationList,
+                window_size,
+                window_flags::flag_19,
```

Comment:
I think this is the wrong flag. Likely 11 instead? But the flag I'm really missing is the resizable flag.